PolyEngine/Core/Src/pe/core/storage/impl/IndexedStringManager.cpp

```diff
@@ -70,7 +70,7 @@ void IndexedStringManager::setTTLMode(bool enabled)
 {
 	if(m_ttlEnabled && !enabled)
 	{
-		DISCARD m_ttlEntries.empty();
+		std::swap(m_ttlEntries, std::priority_queue<TTLEntry>());
```

Comment:
This does not compile because you are trying to swap into a temporary (rvalue) and unless it is called as member function as shown [https://en.wikibooks.org/wiki/More_C%2B%2B_Idioms/Clear-and-minimize](here) it won't.
Why just not call clear on the m_ttlEntries?

Reply:
I believe stl priority queue doesn't have clear method.